src/test/java/reposense/system/CommandRunnerTest.java

```diff
@@ -22,48 +26,63 @@ public void cloneTest() {
     }
 
     @Test
     public void logWithContentTest() {
         String content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, null, null);
-        Assert.assertNotEquals(content.length(), 0);
+        Assert.assertFalse(content.isEmpty());
     }
 
     @Test
     public void logWithoutContentTest() {
-        Calendar c = Calendar.getInstance();
-        c.set(2050, 1, 1, 0, 0);
-        String content = "";
+        Date date = TestUtil.getDate(2050, Calendar.JANUARY, 1);
+        String content;
         try {
-            content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, c.getTime(), null);
-            Assert.assertEquals(content.length(), 0);
+            content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, date, null);
+            Assert.assertTrue(content.isEmpty());
         } catch (Exception e) {
-            Assert.assertTrue(false);
+            fail();
         }
-        c.set(1950, 1, 1, 0, 0);
 
+        date = TestUtil.getDate(1950, Calendar.JANUARY, 1);
         try {
-            content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, null, c.getTime());
-            Assert.assertEquals(content.length(), 0);
+            content = CommandRunner.gitLog(TestConstants.LOCAL_TEST_REPO_ADDRESS, null, date);
+            Assert.assertTrue(content.isEmpty());
         } catch (Exception e) {
-            Assert.assertTrue(false);
+            fail();
         }
-
     }
 
     @Test
     public void blameRawTest() {
+        Date sinceDate = TestUtil.getDate(1950, Calendar.JANUARY, 1);
+        Date untilDate = TestUtil.getDate(2050, Calendar.JANUARY, 1);
         try {
-            String content = CommandRunner.blameRaw(TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java");
-            Assert.assertNotEquals(content.length(), 0);
+            // supply both sinceDate and untilDate -> success
+            String content = CommandRunner.blameRaw(
+                    TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java", sinceDate, untilDate);
+            Assert.assertFalse(content.isEmpty());
+
+            // supply only untilDate -> success
+            content = CommandRunner.blameRaw(
+                    TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java", null, untilDate);
+            Assert.assertFalse(content.isEmpty());
+
+            // supply only since sinceDate -> success
+            content = CommandRunner.blameRaw(
+                    TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java", sinceDate, null);
+            Assert.assertFalse(content.isEmpty());
+
+            // no sinceDate or untilDate supplied -> success
+            content = CommandRunner.blameRaw(
+                    TestConstants.LOCAL_TEST_REPO_ADDRESS, "blameTest.java", null, null);
+            Assert.assertFalse(content.isEmpty());
         } catch (Exception e) {
-            Assert.assertTrue(false);
+            fail();
         }
```

Comment:
Can avoid the try-catch if you add `throws Exception` to the method signature. An exception fails the test anyway.